.travis.yml

```diff
@@ -1,25 +0,0 @@
-dist: trusty
-language: python
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "pypy3"
-matrix:
-  include:
-  - python: 3.8
-    dist: xenial
-  - python: 3.9
-    dist: xenial
-  - python: pypy
-    dist: xenial
-  - python: pypy3
-    dist: xenial
-before_install:
-  - pip install --upgrade pip setuptools
-install:
-  - pip install -r requirements.txt
-  - if [[ $TRAVIS_PYTHON_VERSION != pypy* ]]; then pip install coveralls; fi

```

Comment:
There is no coveralls step in github actions. We should add it before removing travis.